bin/main.rb

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/ruby
+#
+require_relative '../lib/tictactoe'
+
+class MainIO
+  def initialize(stdin: '', stdout: [])
```

Comment:
I like the use of keyword parameters - I've generally found they help me read and understand code better :)